sorc/chgres_cube.fd/input_data.F90

```diff
@@ -6681,4 +6718,226 @@ subroutine check_cnwat(cnwat)
   enddo
 end subroutine check_cnwat
 
+
+
+
+!> Pressure to presure vertical interpolation for tracers with linear or lnP
+!> interpolation. Input tracers on pres levels are interpolated 
+!> to the target output pressure levels. The matching levels of input and 
+!> output will keep the same. Extrapolation is also allowed but needs
+!> caution. The routine is mostly for GFSV16 combined grib2 input when spfh has
+!> missing levels in low and mid troposphere from U/T/HGT/DZDT. 
+!!
+!! @param ppin     [in] 1d input pres levs
+!! @param xxin     [in] 1d input tracer
+!! @param npin     [in] number of input levs 
+!! @param ppout    [in] 1d target pres levs 
+!! @param xxout    [out] 1d interpolated tracer
+!! @param npout    [in] number of target levs 
+!! @param linlog   [in] interp method.1:linear;not 1:log;neg:extrp allowed
+!! @param xmsg     [in] fill values of missing levels (-999.0)
+!! @param ier      [out] error status. non 0: failed interpolation
+!! @author NCL code for pressure level interpolation
```

Comment:
Remove the first @author. 

Do author as follows: @author Jili Dong NCEP/EMC

Reply:
I would add a @date as well.

Reply:
Sorry, I meant lowercase: @date. Otherwise you get a doxygen warning. You will note that one of the checks failed.

Reply:
One more time: `@date`

Reply:
> One more time: `@date`

changed already. But the intel test failed at downloading some index file:

_E: Some index files failed to download. They have been ignored, or old ones used instead.
Error: Process completed with exit code 100._

Don't know how to rerun the test.